application-1/analytics/deployment.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app.kubernetes.io/component: backend
     app.kubernetes.io/instance: user-analytics
     app.kubernetes.io/version: "1.0"
+  annotations:
+    app.openshift.io/vcs-uri: "git@github.com:redhat-developer/topology-example.git"
```

Comment:
Also, do open a PR to fix the typo on app-labels repo, @DhritiShikhar .

Reply:
@sbose78 are you talking about this - https://github.com/gorkem/app-labels/pull/11 ?